<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addservice</class>
 <widget class="QDialog" name="addservice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>781</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ID SERVICE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>TYPE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>PRIX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Ajouter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;background-color:rgb(27, 75, 104);&lt;/p&gt;&lt;p&gt;color:white;&lt;/p&gt;&lt;p&gt;border-radius:15px;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(27, 75, 104);
color:white;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IDSERVICE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PRIX">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>791</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/icons/Inter.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>lavage</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lavage interieur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fast food</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>verifier les pneux</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>soint mecanique</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ajouter un service</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEdit_IDSERVICE</zorder>
   <zorder>lineEdit_PRIX</zorder>
   <zorder>pushButton_Ajouter</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_7</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
